infra/docs.sh

```diff
@@ -16,5 +16,5 @@ cp dev/docs/google2ed1af765c529f57.html dev/docs/doc
 
 # Upload the docs.
 if [ "$1" = "--upload" ]; then
-  gsutil -m rsync -d -r dev/docs/doc/ gs://docs.flutter.io/
+  gsutil -m cp -z "js,json,html,css" dev/docs/doc/ gs://docs.flutter.io/
```

Comment:
Copy isn't good because it will leave orphans.  What we really want is the ability to upload to the server and then atomically switch to the new content.  Maybe we can `cp` to a tmp bucket and then `mv` the bucket into the final location?  That's how you'd do this with a normal file system.